Modules/Quotas/test/Quotas.Domain.Tests/IdentityTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Domain.Tests;
+
+public class IdentityTests
+{
+    [Fact]
+    public void Can_create_identity_with_valid_properties()
+    {
+        var address = TestDataGenerator.CreateRandomBytes().ToString();
```

Comment:
Using a hardcoded value makes it easier to read, in my opinion. 

```suggestion
        var address = "someAddress"
```